airbyte-integrations/connectors/source-shopify/acceptance-test-config.yml

```diff
@@ -13,7 +13,7 @@ tests:
     - config_path: "secrets/config.json"
       # When account doesn't have the Refunds
       configured_catalog_path: "integration_tests/no_refunds_catalog.json"
-      validate_output_from_all_streams: yes
+      validate_schema: yes
```

Comment:
By default `validate_schema` is True. Plz, delete it)

Reply:
Done. Thanks)